### PR TITLE
Bug 1946178: Fix OCS validation message

### PR DESCRIPTION
**Pull request by rewantsoni**
Closes: [OCPBUGSM-27298](https://issues.redhat.com/browse/OCPBUGSM-27298)

Signed-off-by: Rewant Soni <resoni@redhat.com>

# Description
Modify OCS Host and Cluster Validation message to make it clearer.

# What environments does this code impact?

- [ ] Automation (CI, tools, etc)
- [ ] Cloud
- [ ] Operator Managed Deployments
- [x] None

# How was this code tested?

Please, select one or more if needed:

- [ ] assisted-test-infra environment
- [ ] dev-scripts environment
- [ ] Reviewer's test appreciated
- [ ] Waiting for CI to do a full test run
- [ ] Manual (Elaborate on how it was tested)
- [x] No tests needed

# Assignees

/cc @nbalacha 
/cc @priyanka19-98 
/cc @gobindadas

## Checklist

- [x] Title and description added to both, commit and PR.
- [x] Relevant issues have been associated (see [CONTRIBUTING] guide)
- [x] Reviewers have been listed
- [x] This change does not require a documentation update (docstring, `docs`, README, etc)
- [x] Does this change include unit-tests (note that code changes require unit-tests)

## Reviewers Checklist

- [ ] Are the title and description (in both PR and commit) meaningful and clear?
- [ ] Is there a bug required (and linked) for this change?
- [ ] Should this PR be backported?

### CODE REVIEW

File: internal/operators/ocs/validations.go [hunk 1]
```diff
@@ -77,7 +77,7 @@ func (o *operator) computeResourcesAllNodes(cluster *models.Cluster, ocsClusterR
 		*/
 		if !compactMode {
 			if host.Role == models.HostRoleAutoAssign {
-				status = "All host roles must be assigned to enable OCS."
+				status = "For OCS, all host roles must be assigned to worker for Standard Mode."
```
Comment (priyanka19-98) [inline internal/operators/ocs/validations.go:80]: What if the user has 6 hosts with auto-assign mode? The user would consider changing all host roles to worker according to the message.

Comment (rewantsoni) [reply internal/operators/ocs/validations.go:80]: We can reword it to, "For OCS, all host roles must be assigned to master or worker for Standard Mode." WDYT?

**Comment by gobindadas** [comment]
LGTM

**Comment by gamli75** [comment]
@lalon4 @rewantsoni will this wording change require a fix in the e2e tests?

**Comment by rewantsoni** [comment]
> @lalon4 @rewantsoni will this wording change require a fix in the e2e tests?

No this won't require a change in the e2e test

**Review by gamli75 — APPROVED** [review]
